src/autogpt_plugins/baidu_search/baidu_search.py

```diff
@@ -0,0 +1,36 @@
+import requests
+import json
+import os
+import re
+from bs4 import BeautifulSoup
```

Comment:
Can you add this to the requirements.txt so the tests pass

Reply:
Hi, I have updated the requirements and the test has passed